openmetadata-ui/src/main/resources/ui/src/components/TableQueries/table-queries.style.less

```diff
@@ -40,3 +38,23 @@
 .query-card-container.selected {
   border: 1px solid @info-color;
 }
+
+.query-right-panel {
+  position: sticky;
+  height: calc(100vh - @summary-panel-offset);
+  z-index: 9;
```

Comment:
We should not modify zIndex, Is it required?

Reply:
Thanks for pointing this out, and yes we don't need z-index here!